<launch>
    <arg name="debug" default="false"/>
    <arg name="lidar_test_mode" default="false"/>

    <node pkg="bigbang_eurobot" name="bigbang_rplidar" type="bigbang_rplidar" output="screen">
        <param name = "debug" value = "$(arg debug)"/>
        <param name = "test_mode" value = "$(arg lidar_test_mode)"/>
    </node>

    <!-- <node pkg="tf" type="static_transform_publisher" name="static_laser_tf" args="0 0 0 0 0 0 filtered_pos laser 20" /> -->

</launch>
